pkg/storage-init/storage-init.sh

```diff
@@ -148,11 +153,13 @@ if P3=$(findfs PARTLABEL=P3) && [ -n "$P3" ]; then
         fi
     fi
 
+    # deposit fs type into /run
+    echo "$P3_FS_TYPE" > /run/eve.persist_type
```

Comment:
Actually this adds a CR at the end thus AFAICT the readfile+comparison with "zfs" will always fail.

Reply:
Yup -- as was reported by @cshari-zededa as well